engine/apps/chatops_proxy/tests/test_events.py

```diff
@@ -0,0 +1,76 @@
+from unittest.mock import patch
+
+import pytest
+from django.test import override_settings
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APIClient
+
+from apps.chatops_proxy.events import ChatopsEventsHandler
+from apps.chatops_proxy.events.handlers import SlackInstallationHandler
+
+installation_event = {
+    "event_type": "integration_installed",
+    "data": {
+        "provider_type": "slack",
+        "stack_id": "stack_id",
+        "grafana_user_id": "grafana_user_id",
+        "payload": {
+            # It's not actual payload we are getting from slack, just a placeholder
+            "slack_id"
+            "some_slack_id"
```

Comment:
Is this sample data ok ^?

Reply:
Missed a column there, it should be "slack_id": "some_slack_id". Thanks!
I didn't put whole payload here because tests are not using it – install_slack_integration mocked in these tests. I'm testing install_slack_integration on it's own.

Reply:
Thanks for noticing. missed a coma. Anyway, I went ahead and used a more accurate payload example from slack docs instead of placeholder